docs/mcp-elements.md

```diff
@@ -154,31 +154,37 @@ public function getMultipleContent(): array
 
 #### Error Handling
 
-Tools can throw exceptions which are automatically converted to proper JSON-RPC error responses:
+**Tools should ONLY throw `ToolCallException` for any error that occurs during tool execution.**
```

Comment:
that "should ONLY" confused me a bit - especially since we usually have that MoSCoW stuff

For example?
```suggestion
**Tools SHOULD throw `ToolCallException` for errors that occur during tool execution to be communicated to the client.**
```